README.md

```diff
@@ -249,6 +249,12 @@ profile.
 
 - For `initscripts`,  `mac` is the currently configured MAC address of the device (`HWADDR`).
 
+### `mtu`
+
+The `mtu` option denotes the maximum transmission unit for the profile's
+device. Please note, if a profile depends on another profile, the value of the
+`mtu` option cannot be higher than the value for the parent's device `mtu`.
```

Comment:
what about writing it a bit differently:
"Please note, for logical devices (for example those of type `bond` or `vlan`) the value of the `mtu` option cannot be higher than the value for the underlying physical device `mtu`."
The reason is that the relation of dependency is different for `bond` and for `vlan` - in `bond` the slaves (physical devices) refer to (depends on) their `master` (logical device) while for `vlan` the vlan (logical device) refers to (depends on) its physical device.

Reply:
that said, maybe this note makes really sense only for VLANs - can a bond have a different MTU than its slaves?